docs/help/git_webhooks.md

```diff
@@ -0,0 +1,57 @@
+## Webhooks
+
+### Give the bot an SSL-certificate
+
+#### Self-signed certificate
+
+***WARNING!*** Your git host may not accept self-signed certificates.
+
+`openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 3650`
+
+#### [acme.sh](https://github.com/Neilpang/acme.sh) certificate
+
+Assuming it's already installed and configured,
+
+TODO: expand this
+
+```
+$ACMESH --key-file $BITBOTDIR/key.pem --fullchain-file $BITBOTDIR/cert.pem"
+chmod -R 700 $BITBOTDIR/ssl.key
+chown -R bitbot:bitbot $BITBOTDIR
+```
+### Configure the bot
+
+1. Enable the REST API in bot.conf by adjusting
+
+```
+# key/cert used for REST API
+tls-api-key              = /path/to/key.pem
+tls-api-certificate      = /path/to/cert.pem
+api-port                 = 5000
+```
+
+you may change the API port if you wish.
+
+2. restart the bot or send it a `SIGUSR1` signal
```

Comment:
Is this correct? I have no idea how to advice `SIGUSR1`ing the bot so I would like to keep restart here.

Reply:
`$ kill -USR1 <pid>` but then you gotta find the pid etc...

Reply:
Is this current step OK?

Reply:
yes